extension.js

```diff
@@ -123,25 +98,25 @@ function activate(context) {
 
   const regexpWithAllChars = new RegExp(
     Object.keys(gremlinsConfig)
-      .map(hexCodePoint => charFromHex(hexCodePoint) + '+')
+      .map((hexCodePoint) => charFromHex(hexCodePoint) + '+')
       .join('|'),
     'g',
   )
 
   vscode.window.onDidChangeActiveTextEditor(
-    editor => updateDecorations(editor, gremlins, regexpWithAllChars),
+    (editor) => updateDecorations(editor, gremlins, regexpWithAllChars),
     null,
     context.subscriptions,
   )
 
   vscode.window.onDidChangeTextEditorSelection(
-    event => updateDecorations(event.textEditor, gremlins, regexpWithAllChars),
+    (event) => updateDecorations(event.textEditor, gremlins, regexpWithAllChars),
```

Comment:
We use Prettier to format our code, and like to avoid parentheses around a sole arrow function parameter:
https://github.com/nhoizey/vscode-gremlins/blob/master/package.json#L60-L64
https://prettier.io/docs/en/options.html#arrow-function-parentheses

Didn't your IDE use Prettier when saving your edits?